server/routerlicious/packages/services-shared/src/http.ts

```diff
@@ -37,6 +37,17 @@ export function validateRequestParams(...paramNames: (string | number)[]): Reque
 	};
 }
 
+/**
+ * Converts the request param to a boolean
+ */
+export function getBooleanParam(param: any): boolean {
```

Comment:
This is neat. I guess we plan to use this in GitRest/Historian as well whenever a new pre-release/release is available for r11s packages?

Reply:
Yes. That is the plan.